src/engine/game/battle.lua

```diff
@@ -1125,7 +1125,9 @@ function Battle:processAction(action)
                 else
                     dmg_sprite:setScale(2, 2)
                 end
-                dmg_sprite:setPosition(enemy:getRelativePos(enemy.width/2, enemy.height/2))
+                local relative_pos_x, relative_pos_y = enemy:getRelativePos(enemy.width/2, enemy.height/2)
+                local dmg_sprite_offset = enemy.dmg_sprite_offset
+                dmg_sprite:setPosition(relative_pos_x + dmg_sprite_offset[1], relative_pos_y + dmg_sprite_offset[2])
```

Comment:
You could use `enemy.dmg_sprite_offset` directly -- this takes more characters than doing that